fontc/src/args.rs

```diff
@@ -23,6 +25,10 @@ pub struct Args {
     #[arg(short, long)]
     source: Option<PathBuf>,
 
+    #[clap(skip)]
+    #[serde(skip)]
+    pub input_binary: Option<InMemorySource>,
```

Comment:
can we add a line of documentation noting that this is for use by wasm, (or something else, if I have misunderstood)?